src/Service/Daemon.hs

```diff
@@ -492,10 +493,10 @@ run' threadMapTV = do
       where
         mkDefaultTopicMsgAction automationBroadcast' = \_topic topicMsg ->
           for_ (decode topicMsg) $
-              atomically
-            . writeTChan automationBroadcast'
-            . Automation.Client automationName
-            . Automation.ValueMsg
+                Automation.ValueMsg
+            >>> Automation.Client automationName
+            >>> writeTChan automationBroadcast'
+            >>> atomically
```

Comment:
random side tweak, I think it reads a little better in reverse composition order